orchestration/templates/cron-workflow.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: dog-aging-ingest
+spec:
+  suspend: {{ not .Values.cron.enable }}
+  schedule: {{ .Values.cron.schedule | quote }}
+  timezone: {{ .Values.cron.timezone | quote }}
+  concurrencyPolicy: Forbid
+  workflowSpec:
+    entrypoint: main
+    serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+    podGC:
+      strategy: OnWorkflowSuccess
+    templates:
+      ##
+      ## Main entry-point to the regularly-run Dog Aging ingest workflow.
+      ##
+      ## Delegates to a WorkflowTemplate for all of the business logic,
+      ## to support easily running manual ingests.
+      ##
+      - name: main
+        steps:
+          - - name: run-main-template
+              templateRef:
+                name: ingest-dog-aging-archive
```

```suggestion
                name: ingest-dog-aging
```
This should match the name of the script below, right?

---

Yes! Thank you!